### PR TITLE
Post-merge cleanup

### PR DESCRIPTION
**Pull request by hybridcattt**
Accidentally left couple things commented out during merge of #28. Fixing that now + the tag approach was confusing to me, so improved that too.

### CODE REVIEW

File: Say Their Names/Source/Controller/Home/CollectionViewDataSource/PersonCollectionViewDataSource.swift [hunk 1]
```diff
@@ -37,7 +37,6 @@ extension PersonCollectionViewDataSource: UICollectionViewDataSource {
     }
     
     func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
-        if collectionView.tag == 0 { return UICollectionReusableView() }
         let headerView = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: PersonHeaderCell.headerIdentifier, for: indexPath) as! PersonHeaderCell
```
Comment (evilpenguin) [inline Say Their Names/Source/Controller/Home/CollectionViewDataSource/PersonCollectionViewDataSource.swift:40]: Why are we casting here? This isn't needed unless we are doing things with `PersonHeaderCell`

File: Say Their Names/Source/Controller/Home/HomeController.swift [hunk 2]
```diff
@@ -119,42 +120,37 @@ class HomeController: UIViewController, ServiceReferring {
...
-        let width = collectionView.frame.width - 32
-        return collectionView.tag == 0 ? .zero : .init(width: width, height: 170)
+        if collectionView === locationCollectionView {
+            return CGSize.zero
+        }
+        else if collectionView === peopleCollectionView {
+            let width = collectionView.frame.width - 32
+            return CGSize(width: width, height: 170)
+        }
+        else {
```
Comment (evilpenguin) [inline Say Their Names/Source/Controller/Home/HomeController.swift:130]: I know the `else {}` wrapping the final return is readability but not required.

File: Say Their Names/Source/Controller/Home/HomeController.swift [hunk 3]
```diff
@@ -119,42 +120,37 @@ class HomeController: UIViewController, ServiceReferring {
...
-        return collectionView.tag == 0 ? locationCellSize : peopleCellSize
+        
+        if collectionView === locationCollectionView {
+            return CGSize(width: 103, height: 36)
+        }
+        else if collectionView === peopleCollectionView {
+            let width = collectionView.frame.width / 2 - 24
+            return CGSize(width: width, height: 300)
+        }
+        else {
```
Comment (evilpenguin) [inline Say Their Names/Source/Controller/Home/HomeController.swift:144]: Same here.

File: Say Their Names/Source/Controller/Home/HomeController.swift [hunk 4]
```diff
@@ -119,42 +120,37 @@ class HomeController: UIViewController, ServiceReferring {
...
+            return CGSize.zero
+        }
     }
     
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        if collectionView.tag == 0 {
-            // Location CollectionView
-            
-        } else {
+        if collectionView === locationCollectionView {
```
Comment (evilpenguin) [inline Say Their Names/Source/Controller/Home/HomeController.swift:150]: Do we really need code that does nothing? Can we remove this?

Comment (evilpenguin) [reply Say Their Names/Source/Controller/Home/HomeController.swift:150]: Maybe a simple comment for now.

**Comment by evilpenguin** [comment]
Saw comment on email about comment vs imple.

**Comment by hybridcattt** [comment]
Here I changed it so code follows different path depending on which collection view instance it is. And if we add a third collection view, so it wouldn't break. 
If there wasn't a third possible option, then yeah it's just a matter of taste I think :)

**Pull request merged**